src/pdf/ShoppingList/ShoppingListPDF.tsx

```diff
@@ -0,0 +1,236 @@
+"use client";
+
+import React from "react";
+import { Document, Page, Text, View, Image, StyleSheet } from "@react-pdf/renderer";
+import {
+    BlockProps,
+    ClientSummary,
+    Item,
+    ShoppingListPDFProps,
+} from "@/pdf/ShoppingList/dataPreparation";
+
+const styles = StyleSheet.create({
+    paper: {
+        marginLeft: "0.75in",
+        marginRight: "0.75in",
+        lineHeight: "1.5pt",
+    },
+    flexRow: {
+        display: "flex",
+        flexDirection: "row",
+    },
+    flexColumn: {
+        display: "flex",
+        flexDirection: "column",
+        padding: "5pt",
+    },
+    pdfHeader: {
+        justifyContent: "space-between",
+    },
+    logoStyling: {
+        maxWidth: "20%",
+        alignSelf: "center",
+    },
+    infoBlock: {
+        borderStyle: "solid",
+        border: "1pt",
+        borderBottom: "0",
+    },
+    infoCell: {
+        width: "50%",
+        padding: "5pt",
+    },
+    tableRow: {
+        borderStyle: "solid",
+        border: "1pt",
+        paddingVertical: "4pt",
+        textAlign: "center",
+    },
+    tableItemDescription: {
+        width: "25%",
+    },
+    tableQuantity: {
+        width: "35%",
+    },
+    tableNotes: {
+        width: "30%",
+    },
+    tableDone: {
+        width: "10%",
+    },
+    title: {
+        fontSize: "30pt",
+        fontFamily: "Helvetica-Bold",
+    },
+    subtitle: {
+        fontSize: "24pt",
+        fontFamily: "Helvetica",
+    },
+    keyText: {
+        fontSize: "12pt",
+        fontFamily: "Helvetica-Bold",
+    },
+    normalText: {
+        fontSize: "12pt",
+        fontFamily: "Helvetica",
+    },
+    inputText: {
+        paddingTop: "20pt",
+    },
+    checkBox: {
+        alignSelf: "center",
+        width: "18pt",
+        height: "18pt",
+        borderStyle: "solid",
+        border: "1pt",
+    },
+});
+
+const OneLine: React.FC<{ header: string; value: string }> = ({ header, value }) => {
```

Comment:
can you interface all these bits? just needs an interface immediately above this - eg
```ts
interface OneLineProps {
    header: string;
    value: string;
}
const OneLine: React.FC<OneLineProps> = ({header, value}) => {...}
```
and so on for the rest of the file :)